RecoEgamma/ElectronIdentification/interface/ElectronMVAEstimatorRun2Fall17iso.h

```diff
@@ -0,0 +1,16 @@
+#include "RecoEgamma/ElectronIdentification/interface/ElectronMVAEstimatorRun2Fall17.h"
+
+class ElectronMVAEstimatorRun2Fall17iso : public ElectronMVAEstimatorRun2Fall17 {
```

Comment:
it would be better to call it ElectronMVAEstimatorRun2Fall17Iso to consistently use CamelCase.  Likewise for ElectronMVAEstimatorRun2Fall17NoIso